rfc000-rfc-process.md

```diff
@@ -12,6 +12,8 @@ community use RFCs to document, discuss, and plan improvements to Chef and
 other Chef ecosystem tools. You can always find the list of accepted RFCs at
 https://github.com/chef/chef-rfc.
 
+The specific projects covered by this process are detailed in RFC002 (Scope).
```

Comment:
next, now is :beer:. 

This didn't feel like the right document for a long list, especially once I started pondering grouping by exceptions. For instance chef-server and test-kitchen have their own maintainers file. Most the RFCs here are applicable to chef-server, but not so much test-kitchen. However I don't want more than one chef-rfc either.

Obviously early feedback welcome.